packages/amplify-category-auth/tests/commands/enable.test.js

```diff
@@ -25,6 +25,22 @@ describe('auth enable: ', () => {
     expect(add.run).toBeDefined();
   });
 
+  /*
+  describe('case: amplify not initialized', () => {
+    beforeEach(() => {
+      mockGetProjectDetails.mockReturnValue({
+        projectConfig: {},
+        amplifyMeta: {}
+      });
+    });
+
+    it('enable method should detect non existent amplify project and return after printing error text', async () => {
+      await add.run(mockContext);
+      expect(mockContext.print.error).toBeCalledWith("You are not working inside a valid amplify project.\nUse 'amplify init' in the root of your app directory to initialize your project with Amplify");
+    });
+  });
+  */
+
```

Comment:
Did these test cases exist before you started working on this PR? Did you comment them out because your changes make these tests out-of-date? If that's the case, I'm not sure if these tests should be left here as comments, or if these should be deleted.

Reply:
I completely forgot about those! I started working on some draft tests on that file and just commented them out before moving onto the new test file. Pushing these changes right now 🙌